noxfile.py

```diff
@@ -120,7 +128,7 @@ def run_scripts(session):
     # is fixed
     session.install("setuptools", silent=False)
     session.install("-e", ".[all,dev]", silent=False)
-    session.run("python", "run-tests.py", "--scripts")
+    session.run("python", "-m", "pytest", "-svra", "--scripts", "tests/test_scripts.py")
```

Comment:
```suggestion
    session.run("python", "-m", "pytest", "--scripts", "tests/test_scripts.py")
```

Reply:
For example, this session runs `pytest --scripts`, while the `unit` session runs `pytest -m unit` (related to my question above). I think `pytest --unit`, `pytest --integration`, etc. are shorter and better to use.

Also, it would be great if we could configure `pytest --scripts` to run just `tests/test_scripts.py` itself, rather than us having to specify the file.

Reply:
Discussed on Slack, and `pytest -m unit` is the better one to use